rtirl-obs/api Module not found: Error

Module not found: Error: Package path . is not exported from package node_modules\@rtirl\api (see exports field in node_modules\@rtirl\api\package.json)  

Resolve by editing package.json exports as follows:
```
  "exports": {
    ".": "./lib"
  },
  ```

--- a/api/package.json
+++ b/api/package.json
@@ -11,7 +11,9 @@
   "files": [
     "lib/**/*"
   ],
-  "exports": "./lib/",
+  "exports": {
+    ".": "./lib"
+  },
   "author": "muxfdz@gmail.com",
   "license": "MIT",
   "dependencies": {

